web/src/emoji_picker.ts

```diff
@@ -548,10 +612,10 @@ function process_keypress(e) {
     }
 }
 
-export function emoji_select_tab($elt) {
-    const scrolltop = $elt.scrollTop();
+export function emoji_select_tab($elt: JQuery): void {
+    const scrolltop = $elt.scrollTop() ?? 0;
     const scrollheight = $elt.prop("scrollHeight");
-    const elt_height = $elt.height();
+    const elt_height = $elt.height() ?? 0;
```

Comment:
Since `$elt` is not `undefined` as per your type. So `.hieght()` attribute should be defined ??
```
const elt_height = $elt.height() !
```
makes more sense IMO. Same for `scrollTop()`